### PR TITLE
Change guids to be fetched by function instead of in contract

### PR DESCRIPTION
**Pull request by FirestarJes**

### CODE REVIEW

**Review by djorgensendk — DISMISSED** [review]
Approve only regarding .github/workflows/ci.yml

File: source/Energinet.DataHub.MessageHub.Client.Tests/Peek/DataBundleResponseSenderTests.cs [hunk 1]
```diff
@@ -86,10 +87,9 @@ public async Task SendAsync_ValidInput_SendsMessage()
             var requestMock = new DataBundleRequestDto(
                 Guid.NewGuid(),
                 "7E9D048D-F0D8-476D-8739-AAA83284C9C6",
-                "80BB9BB8-CDE8-4C77-BE76-FDC886FD75A3",
-                new[] { Guid.NewGuid(), Guid.NewGuid() });
+                "80BB9BB8-CDE8-4C77-BE76-FDC886FD75A3");
```
Comment (mbusk88) [inline source/Energinet.DataHub.MessageHub.Client.Tests/Peek/DataBundleResponseSenderTests.cs:90]: Should be in a variable

**Review by mbusk88 — DISMISSED** [review]

File: source/Energinet.DataHub.MessageHub.Client.Tests/Storage/BundleRepositoryTests.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                }
+            };
+
+            var databaseResponse = await client.CreateDatabaseIfNotExistsAsync(config.MessageHubDatabaseId).ConfigureAwait(false); // GetDatabase(config.MessageHubDatabaseId);
+            var bundlesResponse = await databaseResponse
+                .Database
+                .CreateContainerIfNotExistsAsync("bundles", "/pk")
+                .ConfigureAwait(true);
+
+            Container container = bundlesResponse.Container;
```
Comment (michaelmajgaard) [inline source/Energinet.DataHub.MessageHub.Client.Tests/Storage/BundleRepositoryTests.cs:76]: Should be `var`

File: source/Energinet.DataHub.MessageHub.Client.Tests/Peek/DataBundleResponseSenderTests.cs (hunk 1 above)
Comment (FirestarJes) [reply source/Energinet.DataHub.MessageHub.Client.Tests/Peek/DataBundleResponseSenderTests.cs:90]: I'm not quite sure what it is you would want to be in a variable?

File: source/Energinet.DataHub.MessageHub.Client.Tests/Storage/BundleRepositoryTests.cs (hunk 2 above)
Comment (FirestarJes) [reply source/Energinet.DataHub.MessageHub.Client.Tests/Storage/BundleRepositoryTests.cs:76]: Fixed

File: source/Energinet.DataHub.MessageHub.Client.Tests/Peek/DataBundleResponseSenderTests.cs (hunk 1 above)
Comment (mbusk88) [reply source/Energinet.DataHub.MessageHub.Client.Tests/Peek/DataBundleResponseSenderTests.cs:90]: I believe I saw the same Guid more than once in the same file, but I can't find it now. So nevermind :)

**Review by mbusk88 — APPROVED** [review]

**Pull request closed without merging**